nemesis/arena_prep.py

```diff
@@ -942,9 +844,9 @@ def auto_phoenix():  # Calm Lands items
     )
 
     memory.main.wait_frames(30)
-    initArray = memory.main.check_ability(ability=0x8002)
-    print("Initiative weapons: ", initArray)
-    if initArray[4]:
+    init_array = memory.main.check_ability(ability=0x8002)
```

Comment:
I did not suggest you go in and convert all of the camel cases to snake cases. I was simply surprised that there were some still laying around. I had thought Shen had gotten to all of them.